vnext/Shared/RuntimeOptions.h

```diff
@@ -34,3 +34,42 @@ const bool __cdecl GetRuntimeOptionBool(const std::string &name) noexcept;
 const std::int32_t __cdecl GetRuntimeOptionInt(const std::string &name) noexcept;
 
 } // namespace Microsoft::React
+
+/// <summary>
+/// Sets a global boolean value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global key</param>
+void __cdecl Microsoft_React_SetRuntimeOptionBool(const char *name, bool value) noexcept;
```

Comment:
@acoates-ms @vmoroz 

Opinions on this naming convention?

Reply:
I am not sure about the name:
- I would void using underscore in the name.
- I am not easy with the term RuntimeOption - it is too generic.
I think in the short term lets go with them 'as-is' - we will learn about their functionality and can get back to have a different name when we add more ABI-safe functions.

Reply:
Renaming from `Microsoft_React_*` to `MicrosoftReact*`